spec/features/visitor/sign_in_spec.rb

```diff
@@ -29,4 +30,10 @@ def sign_in(email, password)
 
     expect(page).to have_content("You have to confirm your email address before continuing.")
   end
+
+  scenario "js object stores Visitor data", js: true do
```

Comment:
Ok, I found your discussion with @VladimirMikhailov in the  https://github.com/fs/rails-base/issues/363

Not sure current implementation with is readable. What if we just leave changes related to the escaping only?